src/interface/obsidian/src/chat_modal.ts

```diff
@@ -379,8 +397,9 @@ export class KhojChatModal extends Modal {
             } else {
                 // If conversation history is cleared successfully, clear chat logs from modal
                 chatBody.innerHTML = "";
-                await this.getChatHistory(chatBody);
-                this.flashStatusInChatInput(result.message);
+                let getChatHistoryStatus = await this.getChatHistory(chatBody);
+                let statusMsg = getChatHistoryStatus ? result.message : "Failed to clear conversation history";
```

Comment:
If `getChatHistoryStatus` is not `true`, then you would say that the conversation history failed to clear. Shouldn't we avoid clearing the `chatBody.innerHTML` in that case? It would be confusing to the user if the operation failed but their UI also cleared.

Reply:
Nice catch, fixed.